client/src/DashBoardSidebar.js

```diff
@@ -0,0 +1,56 @@
+import { useRouteMatch, Link } from 'react-router-dom';
+import profile from './assets/profile.png';
+
+function Dashboard() {
+  let match = useRouteMatch();
+
+  return (
+    <>
+      {/* @TODO nav should be made a component */}
```

Comment:
Kindly remove these two lines, the comment is not needed anymore just like the fragment